meshery

```diff
@@ -62,7 +62,12 @@ unzip ${PWD}/meshery.zip
 
 printf "\nInstalling mesheryctl in /usr/local/bin.\n"
 WHOAMI=$(whoami)
-if mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl ; then
+if [ $OSEXT = "Linux" ] ; then
+  MV_CMD=$(sudo mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl)
```

Comment:
Thank you, @nupur-thakur! It looks like this will do the trick. A slightly more generic approach (that potentially covers other hiccups) is to attempt a `mv`, catching failure on error due to lack of permissions, executing a `sudo mv` thereafter.

Reply:
Okay Sir, I understand what you are saying but As per I know It's compulsory to have sudo with mv command for /usr/local/bin folder in Linux environment. 
But May be in bigger picture your approach is better so should I change the script according to your suggestions?

Reply:
@nupur-thakur I'm a fan of continual, incremental improvement and am inclined to move forward here. 

The rub in doing so is that we won't be addressing the root cause of the issue - permissions. If this issue only ever occurs in `uname`="Linux", then we're ok. To the extent this isn't the case (that permission is an issue in other environments / situations), this issue will rear its head again. Conceptually, if we address the permission issue directly, we won't see the issue again.

Alternatively, we could accept this PR and log an issue to circle back to improve here at a future point (if ever we arrive at that point). I'll let you decide as its not a critical issue and one that we might not ever see again considering the solution you've provided in this PR.

Reply:
@leecalcote Sir, Now I completely understand that we want to address the issue of permission not for a specific environment but overall whereas previously I was looking at the problem as environment specific. 
I will make appropriate changes and create another pull request with updated documents as well.